observability/logging/logging-6.0/log6x-upgrading-to-6.adoc

```diff
@@ -0,0 +1,387 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-upgrading-to-6"]
+= Upgrading to Logging 6.0
+:context: log6x
+
+toc::[]
+
+Logging v6.0 is a significant upgrade from previous releases, achieving several longstanding goals of Cluster Logging:
+
+* Introduction of distinct operators to manage logging components (e.g., collectors, storage, visualization).
+* Removal of support for managed log storage and visualization based on Elastic products (i.e., Elasticsearch, Kibana).
+* Deprecation of the Fluentd log collector implementation.
+* Removal of support for `ClusterLogging.logging.openshift.io` and `ClusterLogForwarder.logging.openshift.io` resources.
+
+[NOTE]
+====
+The *cluster-logging-operator* does not provide an automated upgrade process.
+====
+
+Given the various configurations for log collection, forwarding, and storage, no automated upgrade is provided by the *cluster-logging-operator*. This documentation assists administrators in converting existing `ClusterLogging.logging.openshift.io` and `ClusterLogForwarder.logging.openshift.io` specifications to the new API. Examples of migrated `ClusterLogForwarder.observability.openshift.io` resources for common use cases are included.
+
```

Comment:
If a cluster was configured for log forwarding to an output and now the same cluster is moved to use Logging 6, this will cause log duplications as the new collectors will start from the begining to read all the logs from the first line causing:

 - Big impact on the log storage for the duplicated logs that can be a lot of GB (we can speak depending on the size of the cluster, but we could speak about more than 100G
 -  Big peak/impact on the resources used by the collectors and nodes where running them, by the Log Storage: cpu, memory, storage used and the network

This was the reason of creating this ticket in the past https://issues.redhat.com/browse/OBSDA-540 and also the article https://access.redhat.com/articles/7063405

Reply:
Discussion with support & dev yesterday determined that this will be a new KCS.